cmd/flags.go

```diff
@@ -259,3 +260,11 @@ func retryFlag(cmd *cobra.Command) *cobra.Command {
 	}
 	return cmd
 }
+
+func updateTimeFlags(cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().DurationP(flagThresholdTime, "n", 6*time.Hour, "time before to expiry time to update client")
```

Comment:
```suggestion
	cmd.Flags().Duration(flagThresholdTime, 6*time.Hour, "time before to expiry time to update client")
```
From my past experience, short hand flags that don't have an obvious usage tend to lead to confusion